### PR TITLE
fix: Mobile-responsive Help pages

### PR DESCRIPTION
**Pull request by LulaV14**
This PR improves the mobile-responsive design of the Help pages so components can be shown correctly.

<br/>

### Markdown Page
Before, the menu component for the markdown syntax was overlapping the main container and it was not readable on mobile. 
After, a floating button was added to trigger a Drawer menu so the user can navigate easily through it. 
  vs 

*Another addition was a floating button for all other views except for mobile to scroll to the top. 
Screen Shot 2022-02-15 at 2 22 46 PM

<br/>

### Integrations Page
Before, the container was too small and for that reason the contents were not readable. 
After, same approach was taken to add the floating button to trigger a drawer menu, and changes were made to the main container to show all inner components correctly so the text is readable.
 vs

### CODE REVIEW

File: packages/frontend/src/pages/help-page/components/help-sidebar/help-sidebar.tsx [hunk 1]
```diff
@@ -14,33 +14,46 @@
...
-      flexBasis={{ base: '16rem', md: '20rem', xl: '22rem' }}
-      flexShrink={0}
-      maxWidth={{ base: '16rem', md: '20rem', xl: '26rem' }}
-      ml="1rem"
-      mt="1rem"
-      pt="1rem"
-      {...props}
-    >
-      <Box position="sticky" top="1rem" overflowY="auto" maxHeight="100vh">
+    <>
```
Comment (baumandm) [inline packages/frontend/src/pages/help-page/components/help-sidebar/help-sidebar.tsx:50]: You just have one component here (the `<Box>`), so you don't need to wrap it in `<></>`.

File: packages/frontend/src/pages/help-page/components/help-sidebar/help-sidebar.tsx [hunk 2]
```diff
@@ -78,6 +91,59 @@ export const HelpSidebar = ({ links, ...props }: Props & BoxProps) => {
...
+        isRound={true}
+        position="fixed"
+        width="3rem"
+        height="3rem"
+        bottom="1.5rem"
+        right="1.5rem"
+        variant="frost"
+        boxShadow="2px 2px 3px #555"
+        icon={iconFactoryAs('listTree')}
+        onClick={(e) => {
```
Comment (baumandm) [inline packages/frontend/src/pages/help-page/components/help-sidebar/help-sidebar.tsx:131]: Can this be simplified to `onClick={onOpen}`?

File: packages/frontend/src/pages/help-page/components/integrations-page/integrations-page.tsx [hunk 3]
```diff
@@ -94,7 +94,7 @@ export const IntegrationsPage = () => {
               browser and provide new functionality.
             </Text>
             <Text>Drag any of the following buttons into your browser's toolbar:</Text>
-            <VStack spacing="1rem" align="flex-start" pl="2rem">
+            <VStack spacing="1rem" align="flex-start" pl={{ base: 'none', sm: '2rem' }} w="full" overflow="scroll">
```
Comment (baumandm) [inline packages/frontend/src/pages/help-page/components/integrations-page/integrations-page.tsx:97]: I can't find a reference for `full` as a valid width value.. maybe it should be `auto`?

Comment (LulaV14) [reply packages/frontend/src/pages/help-page/components/integrations-page/integrations-page.tsx:97]: Oh I believe that's from chakra sizes https://chakra-ui.com/docs/theming/theme#sizes setting 100%

**Pull request merged**